src/js/shield_defs.js

```diff
@@ -925,6 +925,10 @@ export function loadShields(shieldImages) {
   };
   shields["US:CA:Business"] = banneredShield(shields["US:CA"], ["BUS"]);
   shields["US:CA:CR"] = pentagonShieldBlueYellow;
+  shields["US:CA:Mendocino"] = roundedRectShield(
```

Comment:
[Changeset 124,460,057](https://www.openstreetmap.org/changeset/124460057) tags two route relations with this tag. I’m OK with going ahead and merging this PR, since we know what the shield will look like, and the tagging is already settled.